neovim_gui/gtk_ui.py

```diff
@@ -115,6 +116,8 @@ def start(self, bridge):
         window.connect('scroll-event', self._gtk_scroll)
         window.connect('focus-in-event', self._gtk_focus_in)
         window.connect('focus-out-event', self._gtk_focus_out)
+        window.modify_bg(Gtk.StateType.NORMAL,
```

Comment:
what difference does this make? The entire window is painted with the neovim background color so this color would never be seen